failover-slots/README.md

```diff
@@ -0,0 +1,24 @@
+# Failover Slots with Postgres 17
+
+This example shows the usage of Postgres 17 failover replication slots with Postgres SQL-based interface to logical replication as well as with Decodable.
+It accompanies the blog post <TODO>.
+
+To run this example, you’ll need to have these things:
+
+* A free Decodable account
+* The Decodable CLI installed on your machine
+* A free ngrok account
+
+Retrieve your auth token from the ngrok web UI and put it into a file _.env_ like so:
+
+```
+NGROK_AUTH_TOKEN=<your token>
```

Comment:
are you going `NGROK_AUTH_TOKEN` or `NGROK_AUTHTOKEN` because you've got the former here and the latter [here](https://github.com/decodableco/examples/pull/31/files#diff-dca6af601fba0b6d5afed3c04eb52d58c153e914340b319f7eac397e0ca69826R19)

Reply:
Raaah, yes, indeed. Must have forgotten to adjust it here after realizing it needs to be the other one when we spoke.